crates/iota-config/src/node.rs

```diff
@@ -1010,15 +1008,13 @@ pub struct KeyPairWithPath {
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, Eq)]
-#[serde_as]
 #[serde(untagged)]
 enum KeyPairLocation {
```

Comment:
Please double check this isn't used in any other context than the configuration

Reply:
Grepped through the whole repo: `KeyPairLocation` is only used as a private member of `KeyPairWithPath`, which is only ever used in the context of the`NodeConfig`.